### ISSUE TITLE
[Install sdk]: Installation completed, but still lack of dependency.

### ISSUE THREAD

**Issue by ghforgaofeng** [php-opencloud/openstack#221]
hi;
I installed the SDK with this command: "composer require php-opencloud/openstack", 
but I still lack dependencies, whether I need to install it manually, or my method is incorrect.
I hope to get help and I will continue to study “composer”。
My English is not good, I hope you can understand.

thanks。

**Comment by ghforgaofeng** [comment]
Well, I've solved the problem. 
But there are still two files that do not exist.
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-
use OpenStack\Common\Resource\ValueResource;
use OpenStack\Networking\v2\Models\Pool;
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-
At the moment, I'm not sure about the severity of the missing file.

**Comment by haphan** [comment]
@ghforgaofeng  Dependencies are managed by composer, by right all you need is  running command `composer require php-opencloud/openstack`.

I am looking into the issue.

**Comment by haphan** [comment]
Seems like those are unused imports

```
   1) openstack/src/Identity/v2/Models/Token.php (no_unused_imports)
   2) openstack/src/Identity/v3/Service.php (no_unused_imports)
   3) openstack/src/Images/v2/Models/Image.php (no_unused_imports)
   4) openstack/src/Common/Transport/RequestSerializer.php (no_unused_imports)
   5) openstack/src/Common/Transport/Middleware.php (no_unused_imports)
   6) openstack/src/Common/Api/OperatorTrait.php (no_unused_imports)
   7) openstack/src/Common/Service/AbstractService.php (no_unused_imports)
   8) openstack/src/Compute/v2/Models/Host.php (no_unused_imports)
   9) openstack/src/Compute/v2/Models/HypervisorStatistic.php (no_unused_imports)```

Will push a fix for this soon

**Comment by haphan** [comment]
Thanks for report @ghforgaofeng. Issue is fixed in #223
